Changes from develop

wallin opened the pull request.

Commit:
Update regexes, fix android detection and add FIR, KOS and WIO operating systems

madejejej committed:
Fix concurrency issue with MemoryCache

mensfeld commented on lib/device_detector/memory_cache.rb:
```diff
@@ -23,19 +23,15 @@ def get(key)
+      return result if result
+
+      value = yield if block_given?
+      set(string_key, value)
```
could you make 2 cases out of it instead of reassigning value?

spiderpug committed:
Update travis ruby versions

spiderpug committed:
Expose os_family method

spiderpug committed:
Rakefile uses new repository name

spiderpug committed:
run tests on all files

spiderpug committed:
Merge branch 'rporrasluc-update-regexes' into develop

closes #50

spiderpug committed:
update device definitions

https://github.com/matomo-org/device-detector/commit/a62f80f1f52d80fe4d270bbc3bc8669ac924617d

closes #44
closes #45
closes #51
closes #52

spiderpug committed:
remove devices matched by incompatible regex

ruby does not support these non-fixed-length look-behinds

affects:

- impression tabs
- ergo tabs
- sigma x-style tabs

spiderpug committed:
allow memory_cache to store nil values properly

closes #54, closes #55

spiderpug committed:
Merge branch 'fix-concurrency-issue' into develop

spiderpug committed:
Version 1.0.2
